client/components/marketing-survey/cancel-purchase-form/step-components/downgrade-step.jsx

```diff
@@ -0,0 +1,73 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getSelectedSite } from 'state/ui/selectors';
+import { recordTracksEvent } from 'state/analytics/actions';
+import FormSectionHeading from 'components/forms/form-section-heading';
+import FormFieldset from 'components/forms/form-fieldset';
+
+export class DowngradeStep extends Component {
+	static propTypes = {
+		recordTracksEvent: PropTypes.func.isRequired,
+		refundAmount: PropTypes.string,
+		planCost: PropTypes.string,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		translate: noop,
+	};
+
+	onClick = () => {
+		this.props.recordTracksEvent( 'calypso_cancellation_downgrade_step_click' );
+	};
+
+	render() {
+		const { translate, refundAmount, planCost, currencySymbol } = this.props;
+		const amount = currencySymbol + ( refundAmount ? refundAmount : planCost );
+
+		return (
+			<div>
+				<FormSectionHeading>
+					{ translate( 'Would you rather switch to a lower-tier plan?' ) }
+				</FormSectionHeading>
+				<FormFieldset>
+					<p>
+						{ refundAmount
+							? translate(
```

Comment:
`refundAmount` was `0.00` when I tested a non-refundable case, so this check didn't work correctly, and I saw a "You can downgrade to Personal and get a partial refund of $0.00" message rather than the "the new plan will renew at only $X" one.

(Fixing this would also require fixing the missing `planCost` property that @southp noted earlier, since that's used in the second message.)